samples/javascript/E2_GetFileList/index.js

```diff
@@ -1,6 +1,6 @@
 const readdirp = require("readdirp");
```

Comment:
looks like readdirp is also sync only :(

Reply:
😞 This is why we can't have nice syntax.